e2e-tests/commands/insurance/eligibility/forms.js

```diff
@@ -34,6 +34,10 @@ export const completeCompaniesHouseNumberForm = () => {
   selectRadioAndSubmit(FIELD_VALUES.YES);
 };
 
+export const completeCompanyDetailsForm = () => {
```

Comment:
It follows the pattern as other eligibility forms above in this file - I think it's a lot clearer than having `submitButton().click()` throughout various tests; Having a function like this has more context. Also if the form changes (e.g need to do something other than submitting), it can be updated in just one place.